app/models/like.rb

```diff
@@ -1,4 +1,13 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_likes_counter
+    post = Post.find(post_id)
+    if post.likes_counter
+      post.update(likes_counter: post.likes_counter + 1)
+    else
+      post.update(likes_counter: 1)
+    end
+  end
```

Comment:
- [ ] **[OPTIONAL]** We can make this method shorter and also add a callback for it.